morpheus/utils/type_utils.py

```diff
@@ -116,6 +116,9 @@ def pretty_print_type_name(t: typing.Type) -> str:
     Determines a good label to use for a type. Keeps the strings shorter.
     """
 
+    if t is None:
+        return ""
+
```

Comment:
```suggestion
    if t is None:
        return "NoneType"

```

Not sure if there is a reason to return an empty string, but something more descriptive seems like a good idea if its not problematic.

Reply:
I added this bit to get the `Pipeline.visualize` method working for the sole purpose of updating the pipeline graphs to reflect the current state of the examples (ie. dropping the Buffer & Delay stages).

It looks like in the past this displayed the message type between stages, so I just hacked this bit in. Returning a "NoneType" string would cause that to be printed between stages.

Reply:
Gotcha. Well, my vote would be to have something between nodes rather than nothing.

Reply:
Maybe a "->"?
Otherwise I could probably just spend a little more time debugging the visualize code, my guess is it just got broken when the Messages were refactored

Reply:
Split this one off into its own issue #188

Reply:
I reverted this change. The ultimate source of the problem is that the Morpheus pipeline doesn't have access to the type information of the messages between nodes until the pipeline is built, which happens asynchronously, without a way to wait for completion. 

The visualize method gets called early before the pipeline starts so its a race condition between it and the completion of the of the pipeline build. When this happens the `visualize` method gets values of `None`